include/boost/fixed_string/detail/fixed_string.hpp

```diff
@@ -28,6 +29,96 @@ using is_input_iterator =
     std::integral_constant<bool,
         ! std::is_integral<T>::value>;
 
+template<std::size_t N>
+struct priority : priority<N - 1> { };
+
+template<>
+struct priority<0> { };
+
+template<std::size_t N, std::enable_if_t<N < std::numeric_limits<unsigned char>::max()>* = nullptr>
```

Comment:
is `std::enable_if_t` C++11?

Reply:
@vinniefalco Nope, totally forgot about that